src/python_autocite/__main__.py

```diff
@@ -81,27 +81,35 @@ def main():
                     citations.append(soup_to_citation(line, soup))
                 else:
                     print("Unable to load " + str(args.url), file=sys.stderr)
+            print()
+            # Start a new line, split with dot(s)
```

Comment:
Generally, I recommend putting the comment before the line it pertains to, or on the same line, so either:
```
# Start a new line, split with dot(s)
print()
```

OR

```
print() # Start a new line, split with dot(s)
```